tests/framework/extensions/customresourcedefinitions/delete.go

```diff
@@ -0,0 +1,45 @@
+package customresourcedefinitions
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// deletes a single custom resource definition by name
+func DeleteCustomResourceDefinition(client *rancher.Client, clusterID string, namespace string, name string) error {
+	dynamicClient, err := client.GetDownStreamClusterClient(clusterID)
+	if err != nil {
+		return err
+	}
+
+	customResourceDefinitionResource := dynamicClient.Resource(CustomResourceDefinitions).Namespace(namespace)
+	_, err = customResourceDefinitionResource.List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	err = customResourceDefinitionResource.Delete(context.TODO(), name, metav1.DeleteOptions{})
+
+	return err
+}
+
+// deletes a list of custom resource definitions by name
+func BatchDeleteCustomResourceDefinition(client *rancher.Client, clusterID string, namespace string, list []string) error {
+	dynamicClient, err := client.GetDownStreamClusterClient(clusterID)
+	if err != nil {
+		return err
+	}
+
+	customResourceDefinitionResource := dynamicClient.Resource(CustomResourceDefinitions).Namespace(namespace)
+	_, err = customResourceDefinitionResource.List(context.TODO(), metav1.ListOptions{})
```

Comment:
Same question here. Additionally, missing an error return if the lists are required.

Reply:
Yeah I have the same question what purpose does this list serve?

Reply:
They've both been removed